pom.xml

```diff
@@ -535,6 +535,11 @@
             <version>2.15.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Please don't introduce new dependencies for production code. Test-scoped is okay.